src/react/ui-elements/Veeam/VeeamConfiguration.tsx

```diff
@@ -0,0 +1,205 @@
+import Joi from '@hapi/joi';
+import { joiResolver } from '@hookform/resolvers/joi';
+import { Form, FormGroup, FormSection, Stack, Toggle } from '@scality/core-ui';
+import { Button, Input, Select } from '@scality/core-ui/dist/next';
+import { useRef } from 'react';
+import { Controller, useForm } from 'react-hook-form';
+
+const schema = Joi.object({
+  name: Joi.string().required(),
+  version: Joi.string().required(),
+  capacity: Joi.number().required().min(1).max(1023),
+  capacityUnit: Joi.string().required(),
+  enableImmutableBackup: Joi.boolean().required(),
+});
+
+type VeeamConfiguration = {
+  name: string;
+  version: string;
+  capacity: string;
+  capacityUnit: string;
+  enableImmutableBackup: boolean;
+};
+
+const Configuration = () => {
+  const {
+    handleSubmit,
+    control,
+    register,
+    formState: { errors, isValid },
+  } = useForm<VeeamConfiguration>({
+    mode: 'all',
+    resolver: joiResolver(schema),
+    defaultValues: {
+      name: '',
+      version: 'Veeam12',
```

Comment:
nit: why not setting it directly to `Veeam 12`  with the space ? This would avoid later on on the bucket page to do a mapping 'Veeam12' -> 'Veeam 12' 'Veeam11' -> 'Veeam 11'